### ISSUE TITLE
I am having an issue pushing to master

### ISSUE THREAD

**Issue by novakw267** [ga-wdi-boston/full-stack-project#667]
So I had an issue before I dived into my project where I messed up my .gitignore and I think in the process of fixing that I messed my master file I have a separate branch with all my information pushed and back up. I am getting this error trying to push my master after merging to a safe branch with working parts. 

```
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'git@github.com:novakw267/forum-project-backend.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
``` 
Is there a way to fix this before having to stop and make another repository, without having to copy and paste all my code?

**Comment by raq929** [comment]
Yeah, you can't make a new repository, because then you will not meet expectations. (That is, in fact, exactly what the requirement is preventing.) Come see me, I'm near frontlines.

**Comment by novakw267** [comment]
I just got out of my outcomes 1 on 1 are you still at frontlines?

**Comment by BenGitsCode** [comment]
Have you tried understanding why you would get the error you got, and what it means? 
`error: failed to push some refs to <your repo>`

Also, have you read through the hints and tried what git suggests?

**Comment by novakw267** [comment]
We figured it out, I had messed something up and had a commit on the repo that didn't exist locally. We double checked that it wasn't that big of a commit and double checked where I was on my local commit, and decided it was not going to break anything to force the push. 

I had the back end at a workable spot so I split into a back up branch and merged the other branch to master and tried to push it to my repo that's how I found that I had this issue. 

Long story short we force pushed it and it fixed the issue after cleaning up a merger issue. Don't force push without an adult(aka an instructor). 

Also I mean I had an inclining of what was wrong but since it was git I didn't want to do anything wrong and lose a bunch of my data.
